tests/ldevnet.go

```diff
@@ -37,6 +39,28 @@ func LaunchDevnetDocker(t *testing.T, numMiners int) *dockertest.Resource {
 			panic(fmt.Sprintf("couldn't purge lotus-devnet from docker pool: %s", err))
 		}
 	})
+	debug := false
```

Comment:
Leaving this also. It allows to interleave the docker container runs on stdout. By default I left it disabled, but it's handy for looking when things break.